x-pack/plugins/fleet/server/saved_objects/migrations/to_v7_13_0.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectMigrationFn } from 'kibana/server';
+
+import type { Settings } from '../../types';
+
+export const migrateSettingsToV7130: SavedObjectMigrationFn<
+  Exclude<Settings, 'kibana_urls'> & {
```

Comment:
I don't think we need to exclude this right?
```suggestion
  Settings & {
```